ssl-impl/src/main/java/io/crate/protocols/postgres/SslReqConfiguringHandler.java

```diff
@@ -44,7 +44,7 @@ public SslReqConfiguringHandler(Settings settings, SslContext sslContext) {
         this.LOGGER = Loggers.getLogger(SslReqRejectingHandler.class, settings);
```

Comment:
While you're at it, you can also correct this logging class to `SslReqConfiguringHandler.class` 🎉